.github/workflows/ci.yml

```diff
@@ -523,22 +476,26 @@ jobs:
           path: |
             **/surefire-reports/TEST-*.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
+      - name: Update PR check
+        uses: ./.github/actions/update-check
+        if: >-
+          github.event_name == 'repository_dispatch' &&
+          github.event.client_payload.slash_command.sha != '' &&
+          contains(github.event.client_payload.pull_request.head.sha, github.event.client_payload.slash_command.sha)
+        with:
+          pull_request_number: ${{ github.event.client_payload.pull_request.number }}
+          check_name: ${{ github.job }} with secrets
```

Comment:
I was just comparing this to the `Upload test report` section. Should we have `env.ARTIFACT_NAME` in the check_name like we do in the test result name?

Reply:
Good catch, `github.job` is just the `job_id`, so that would be `test`, without the matrix values.

Reply:
@findepi asked also to add a comment to the original PR linking back to this run